content/en/docs/ops/performance-and-scalability/index.md

```diff
@@ -127,8 +127,8 @@ This will decrease the amount data flowing through the system, which will in tur
 - `baseline` Client pod directly calls the server pod, no sidecars are present.
 - `server-sidecar` Only server sidecar is present.
 - `both-sidecars` Client and server sidecars are present. This is a default case inside the mesh.
-- `nomixer-both` Same as **both-sidecars** without Mixer. `MixerV2` latency profile will be similar.
-- `nomixer-server` Same as **server-sidecar** without Mixer. `MixerV2` latency profile will be similar.
+- `nomixer-both` Same as **both-sidecars** without Mixer. `TelemetryV2` latency profile will be similar.
```

Comment:
1. Pl Add telemetry v2 option, mention that target is no-mixer both.
2. Remove the telemetry v2 mention from this and next bullet.